py/README.rst

```diff
@@ -83,30 +83,30 @@ For running on YARN and other supported platforms please see `Running Sparkling
 
       from pysparkling import *
       import h2o
-      hc = H2OContext(sc).start()
+      hc = H2OContext.getOrCreate(sc)
 
 
 Run IPython Notebook with PySparkling
 -------------------------------------
 .. code-block:: bash
 
-    IPYTHON_OPTS="notebook" bin/pysparkling
+    IPYTHON_OPTS="notebook"
```

Comment:
This need to be together on the same line because of bash:
so it can be:
```
 IPYTHON_OPTS="notebook" bin/pysparkling
```
or 
```
export IPYTHON_OPTS="notebook" 
bin/pysparkling
```